lib/ClimaLandSimulations/src/fluxnet/US-MOz/US-MOz_parameters.jl

```diff
@@ -1,123 +1,125 @@
-# DAMM
-α_sx = FT(194e3)
-Ea_sx = FT(61e3)
-kM_sx = FT(5e-3)
-kM_o2 = FT(0.004)
-O2_a = FT(0.209)
-D_oa = FT(1.67)
-p_sx = FT(0.024)
-
-# Autotrophic respiration parameters
-ne = FT(8 * 1e-4)
-ηsl = FT(0.01)
-σl = FT(0.05)
-μr = FT(1.0)
-μs = FT(0.1)
-f1 = FT(0.012)
-f2 = FT(0.25)
-
-# Soil parameters
-soil_ν = FT(0.5) # m3/m3
-soil_K_sat = FT(4e-7) # m/s, matches Natan
-soil_S_s = FT(1e-3) # 1/m, guess
-soil_vg_n = FT(2.05) # unitless
-soil_vg_α = FT(0.04) # inverse meters
-θ_r = FT(0.067) # m3/m3, from Wang et al. 2021 https://doi.org/10.5194/gmd-14-6741-2021
-
-# Soil heat transfer parameters; not needed for hydrology only test
-ν_ss_quartz = FT(0.1)
-ν_ss_om = FT(0.1)
-ν_ss_gravel = FT(0.0);
-κ_quartz = FT(7.7) # W/m/K
-κ_minerals = FT(2.5) # W/m/K
-κ_om = FT(0.25) # W/m/K
-κ_liq = FT(0.57) # W/m/K
-κ_ice = FT(2.29) # W/m/K
-κ_air = FT(0.025); #W/m/K
-ρp = FT(2700); # kg/m^3
-κ_solid = Soil.κ_solid(ν_ss_om, ν_ss_quartz, κ_om, κ_quartz, κ_minerals)
-κ_dry_val = Soil.κ_dry(ρp, soil_ν, κ_solid, κ_air)
-κ_sat_frozen_val = Soil.κ_sat_frozen(κ_solid, soil_ν, κ_ice)
-κ_sat_unfrozen_val = Soil.κ_sat_unfrozen(κ_solid, soil_ν, κ_liq);
-ρc_ds = FT((1 - soil_ν) * 4e6); # J/m^3/K
-z_0m_soil = FT(0.01)
-z_0b_soil = FT(0.001)
-soil_ϵ = FT(0.98)
-soil_α_PAR = FT(0.2)
-soil_α_NIR = FT(0.2)
-
-# TwoStreamModel parameters
-Ω = FT(0.69)
-ld = FT(0.5)
-α_PAR_leaf = FT(0.1)
-λ_γ_PAR = FT(5e-7)
-λ_γ_NIR = FT(1.65e-6)
-τ_PAR_leaf = FT(0.05)
-α_NIR_leaf = FT(0.45)
-τ_NIR_leaf = FT(0.25)
-n_layers = UInt64(20)
-ϵ_canopy = FT(0.97)
-
-# Energy Balance model
-ac_canopy = FT(2.5e3)
-
-# Conductance Model
-g1 = FT(141) # Wang et al: 141 sqrt(Pa) for Medlyn model; Natan used 300.
-Drel = FT(1.6)
-g0 = FT(1e-4)
-
-#Photosynthesis model
-oi = FT(0.209)
-ϕ = FT(0.6)
-θj = FT(0.9)
-f = FT(0.015)
-sc = FT(2e-6) # Bonan's book: range of 2-5e-6
-pc = FT(-2e6) # Bonan's book: -2e6
-Vcmax25 = FT(9e-5) # from Yujie's paper 4.5e-5 , Natan used 9e-5
-Γstar25 = FT(4.275e-5)
-Kc25 = FT(4.049e-4)
-Ko25 = FT(0.2874)
-To = FT(298.15)
-ΔHkc = FT(79430)
-ΔHko = FT(36380)
-ΔHVcmax = FT(58520)
-ΔHΓstar = FT(37830)
-ΔHJmax = FT(43540)
-ΔHRd = FT(46390)
-
-# Plant Hydraulics and general plant parameters
-SAI = FT(1.0) # m2/m2 or: estimated from Wang et al, FT(0.00242) ?
-f_root_to_shoot = FT(3.5)
-K_sat_plant = 5e-9 # m/s # seems much too small?
-ψ63 = FT(-4 / 0.0098) # / MPa to m, Holtzman's original parameter value is -4 MPa
-Weibull_param = FT(4) # unitless, Holtzman's original c param value
-a = FT(0.05 * 0.0098) # Holtzman's original parameter for the bulk modulus of elasticity
-conductivity_model =
-    PlantHydraulics.Weibull{FT}(K_sat_plant, ψ63, Weibull_param)
-retention_model = PlantHydraulics.LinearRetentionCurve{FT}(a)
-capacity = FT(10) # kg/m^2
-plant_S_s = FT(1e-2 * 0.0098) # m3/m3/MPa to m3/m3/m
-rooting_depth = FT(0.5) # from Natan
-z0_m = FT(0.13) * h_canopy
-z0_b = FT(0.1) * z0_m
+"""
+Site-specific model parameters for running CliMA LSM on the Ozark
+fluxtower site.
+"""
```

Comment:
If we want to be able to support prescribing parameters/model type for one model and not for others, we may want to separate the prescription of each internal NamedTuple and then have a function that constructs them all together into the outer NamedTuple. So for example we could have variables for `default_hetero_resp`, `default_auto_resp`, etc., and then have a function that takes in a param set for each model and constructs them into an outer one.

Alternatively we could just overwrite the defaults inside the outer NamedTuple after we build the whole thing for whichever we want to prescribe special parameters for.

Reply:
Or even possibly we could construct the different default params for each model in their own NT, and then in `site_parameters.jl` build each parameters struct from its own param NT and then we could just swap the default for a user defined NT there.

Reply:
Thanks!! These are great suggestions. Let's try it.